site/blog/tutorials/iasql-on-ec2.mdx

```diff
@@ -203,3 +203,138 @@ WHERE id = (SELECT id
+- `iasql_begin`: It'll stop the default cron behavior. Therefore, your changes won't be pushed to the cloud until you manually execute `iasql_commit`. This can be used to "batch" the changes and maybe review them (using `iasql_preview`) before applying. Or rolling back the whole batch (by `iasql_rollback`).
+- `iasql_commit`: Immediately apply the changes from the database to the cloud, and sync the state of the cloud back into the database afterwards.
+
+So why do we have those functions around our insert command? Because we don't want to wait for 2 minutes until the engine decides to push our changes and want them NOW! Why do we want them now? Maybe because we're just impatient, but also because we're going to call an RPC that needs the hosted zone configured for it to work properly.
+:::
+
+Now that we have created our hosted zone, we need to set our domain's nameservers to the ones that Route53 provides us. For each public hosted zone that you create, Amazon Route53 [automatically creates](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/SOA-NSrecords.html) a name server (NS) record and a start of authority (SOA) record. So we should log into the domain registrar, and set the name server values to the four name server values that Route53 has set. To get those values, we can simply run a `SELECT` query:
+```sql
+SELECT record as name_server
+FROM resource_record_set
+WHERE parent_hosted_zone_id = (SELECT id FROM hosted_zone WHERE domain_name = '<your-domain>')
+  AND record_type = 'NS';
+```
+After updating the name servers in the domain settings, we need to wait for those values to be propagated. This can happen in minutes, or even up to 48 hours depending on the registrar.
+
+#### Issue An SSL Certificate For Our Domain
+Having the domain setup in Route53, it's easy to use IaSQL's `certificate_request` RPC to issue a new certificate for it. If you try to go with this procedure through AWS console, you'll need to manually add the CNAME records from the AWS ACM to your hosted zone. But this RPC is implemented in a way that automagically [sets the DNS records](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L177-L188), [waits for the certificate to be issued](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L197), and then [removes those validation DNS records](https://github.com/iasql/iasql/blob/ed1b1ef92ceaa92e7345449992c35a56b62fe571/src/modules/aws_acm/rpcs/request.ts#L200).
```

Comment:
why are you pointing to that specific commit there, instead of pointing to main?

Reply:
Because the `main` branch is subject to change but these links are permalinks that are valid even when the code on `main` changes.

Reply:
Could we change it to one of the versioned tags, though? Linking to `v0.1.6` is a less wonky URL and also includes some information about how "old" these links are relative to whatever the latest release is at the time the user reads this.